### ISSUE TITLE
Remove/fix unfinished content for v3.2

### ISSUE THREAD

**Issue by Yourname942** [ejektaflex/Bountiful#99]
## Bug Report

**Observed Behaviour**
2 error types occured at these lines (read latest log for more details):
```
1283 [14:43:41] [Server thread/ERROR]: Couldn't parse loot table bountiful:blocks/bountyboard
com.google.gson.JsonSyntaxException: Missing ops, expected to find a JsonArray
```
```
1357 [14:43:41] [Server thread/ERROR]: Parsing error loading custom advancement bountiful:main/bounty_complete_rare: Invalid criterion trigger: bountiful:bounty_complete_rare
1358 [14:43:41] [Server thread/ERROR]: Parsing error loading custom advancement bountiful:main/bounty_complete_epic: Invalid criterion trigger: bountiful:bounty_complete_epic
1359 [14:43:41] [Server thread/ERROR]: Parsing error loading custom advancement bountiful:main/bounty_complete_common: Invalid criterion trigger: bountiful:bounty_complete_common
```
```
1362 [14:43:41] [Server thread/ERROR]: Parsing error loading custom advancement bountiful:main/bounty_complete_uncommon: Invalid criterion trigger: bountiful:bounty_complete_uncommon
1363 [14:43:41] [Server thread/ERROR]: Parsing error loading custom advancement bountiful:main/royal_mandate: Invalid criterion trigger: bountiful:royal_mandate
```
**Latest log:** https://gist.github.com/Yourname942/3e6b5038c08c8b683ee4841b3b43e062

**Expected Behaviour**
There should be no errors from your mod when all I did was create a new world and enter it.

**Minimal setup needed to reproduce**
- Minecraft: 1.15.2
- Forge version: 31.2.36
- Bountiful version: 3.1.2
- Kottle: 1.5.2

**Steps to reproduce**
1. Select Play in Twitch App
2. Select Play in Minecraft Launcher
3. Create a new world
4. Enter world
5. Exit Minecraft

**Comment by ejektaflex** [comment]
This is a duplicate of #73 and #76. These are related to incomplete Bountiful features for the next version, and do not affect gameplay in any form as of the current releases. Thank you for the report, though, I appreciate it!

**Comment by ejektaflex** [comment]
I'm going to remove the code that causes these errors to occur with the next release, for 1.16.2. Nothing should change, though 👍

**Comment by ejektaflex** [comment]
I wasn't able to remove the 'missing ops' error - but it's only cosmetic, and won't affect gameplay whatsoever.

**Comment by ejektaflex** [comment]
Fixed in Bountiful 3.3.1 with commit 81dff72! Besides the missing ops, which I will not be fixing.
